docs/implementation-guide.md

```diff
@@ -57,65 +57,49 @@ In this section we describe how a STLT can take this repository and use it to sp
 
 ### User Assumptions
 The steps below assume that a user:
-1. Has some familiarity working in a command line environment on their local machine.
+1. Has some familiarity working in a command line environment.
 2. Has a basic familiarity with Git, and has a GitHub account.
```

Comment:
I believe the QS script now has functionality to handle all required git commands, for the initial repo set up and deploy to GCP. I believe we can drop this assumption for the IG. Now, realistically if users are going to tweak the pipeline for their own needs they'll need git. Eventually we'll probably need a doc about making changes to the pipeline and Git will probably be required for that.

Reply:
Ah nice, will pull this line out!